### ISSUE TITLE
[Windows] Crash when extensions access safebrowsing prefs 

### ISSUE THREAD

**Issue by squalus** [Eloston/ungoogled-chromium#395]
Version: 65.0.3325.181-2

Either of these two lines cause the entire browser to crash when run by an extension:

```
chrome.privacy.services.safeBrowsingEnabled.get({}, function(x){})
chrome.privacy.services.safeBrowsingExtendedReportingEnabled.get({}, function(x){})
```

Maybe related to the changes in `windows-fix-building-without-safebrowsing.patch`

Issue was originally found by @ribatamu in https://github.com/Manvel/Privacy-Manager extension

**Comment by Eloston** [comment]
I'm not getting this in my Debian v65 build, so it sounds more likely to be a Windows-only bug.

**Comment by intika** [comment]
duplicate of https://github.com/Eloston/ungoogled-chromium/issues/619 ?
@Eloston

**Comment by Eloston** [comment]
@intika No. This is a long-standing issue with the patches even in the latest Windows version available.

**Comment by Eloston** [comment]
This should be fixed with 6cc874cfc8a250224811584f2c293dc61ec4faeb.
